src/App.css

```diff
@@ -3,17 +3,114 @@ body {
   padding: 2rem;
 }
 
-li {
-  list-style: none;
-  padding: 0 0.5rem;
+header {
+  display: flex;
+  border: 1px solid #888;
+}
+
+#ul {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 25px;
+  margin-top: 30px;
+}
+
+#linkOne,
+#linkTwo {
+  margin-left: 70px;
+  list-style-type: none;
+  text-decoration: none;
 }
 
-header,
-form,
-nav > ul,
-.bookStyle {
+form {
   display: flex;
   justify-content: space-around;
   align-items: center;
   background-color: #f0f0f0;
+  list-style-type: none;
+}
+
+.book {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-left: 20px;
+  padding-right: 20px;
+  border: 1px solid #888;
+}
+
+.bookDetails {
+  margin: 0;
+  padding: 0;
+  margin-bottom: 15px;
+}
+
+.bookDetails > p {
+  margin: 0;
+  padding: 0;
+  margin-top: 10px;
+}
```

Comment:
- [ ] You have done well by making some changes but just as the previous reviewer suggested, the project still needs some changes in order to look like the Zeplin design guideline. Please take note of font colors, buttons, margins, shadows etc. Please take a look at the attached images for more information 👇 
## Zeplin Design
![image](https://user-images.githubusercontent.com/46839548/201493712-3b92ffd5-6022-43b1-b73c-a17734c51c99.png)

## Your Design
![image](https://user-images.githubusercontent.com/46839548/201493667-62d99bb0-6be7-4df5-b085-9e66bcc9a3bc.png)